### ISSUE TITLE
size in config file is 512x512， train size is 544x544？

### ISSUE THREAD

**Issue by hyaihjq** [gakkiri/simple-centernet-pytorch#3]

**Comment by gakkiri** [comment]
In fact, multi-scale input is adopted.  
So the 544x544 is the largest size in each batch due to the padding in ```collate_fn()```, it's a coincidence.  
If you use batch_size=1, you will find that the input size is different for each batch.  
I think maybe my config parameter is a little misleading.
